docs/get_float.adoc

```diff
@@ -11,27 +11,44 @@ get_float - prompts user for a line of text from stdin and returns the equivalen
 
 *#include <cs50.h>*
 
-*float get_float(string prompt);*
+*float get_float(const string format, ...);*
```

Comment:
Hm, I'm torn to be honest. Odds are, if/when a student looks at `cs50.h`, they'll already have learned what `char *` is. So we could just leave them as `const char *`. And then in reference50, we can simplify to `string`?